### ISSUE TITLE
Custom DDNS invalid URL

### ISSUE THREAD

**Issue by echoDaveD** [Jackysi/advancedtomato#367]
Hello,

i have an ddns address hosted by all-inkl.
So i choose "Custom URL" for DDNS but advancedTomato says the URL is invalid.

The Updateurl is: http://USERNAME:PASSWD@dyndns.kasserver.com.
If i curl the url in Ubuntu Terminal or call it in Firefox it works fine, the ddns entry is refreshing.

Why does AdvancedTomato say the URL is invalid?

best regards,

David

**Comment by Jackysi** [comment]
I don't think that Tomato allows http AUTH within customized URL. But I will check how DDNS is implemented and if that is possible.
